test/acceptance/features/bindAppToService.feature

```diff
@@ -585,3 +585,33 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
         And Secret "binding-request-backend-a" contains "BACKEND_READY" key with value "true"
         And Secret "binding-request-backend-a" contains "BACKEND_HOST" key with value "example.common"
+
+        @negative
+        Scenario: Create Service Binding with empty services
```

Comment:
you cannot create it, as you show later on. Hence, the scenario should reflect that, something like:

"Service Binding with empty services is not allowed in the cluster"

Reply:
Ahh, you got a point here, I'll update it.

Reply:
Done.